lib/kino_db/sql_cell.ex

```diff
@@ -241,6 +238,28 @@ defmodule KinoDB.SQLCell do
     to_req_quoted(attrs, fn _n -> "?" end, :athena)
   end
 
+  # ClichHouse
+  defp to_quoted(%{"connection" => %{"type" => "clickhouse"}} = attrs) do
+    {query, params} =
+      parameterize(attrs["query"], attrs["connection"]["type"], fn _n, ident ->
+        "{#{ident}:String}"
+      end)
+
+    dbg(query)
+    dbg(params)
+
+    quote do
+      {:clickhouse, req_ch_opts} = unquote(quoted_var(attrs["connection"]["variable"]))
```

Comment:
Do we need to store it as a `clickhouse` tuple? It seems it is used to detect if we are using clickhouse, maybe we can add some option or store something in `req` just for internal control? Maybe something in the options?

Reply:
Yeah, I did this because I didn't want to use `ReqCH.new/1` + `Req.post/1` for this, since the API is much simpler  to use directly with `ReqCH.query!/3` - specially because of the query params.

I think I will add an option, and remove later in the process.

Another solution would have a `ReqCH.query(Req.t(), String.t() :: query, [] :: params)` function, without any additional option. But I guess we don't want that API receiving a `req`, right?